src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+$width: 200px;
+
+* {
+  background: $white;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  background: #fff;
+}
+
+.card {
+  display: flex;
+  width: $width;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background: url(../images/star.svg) center / contain no-repeat;
+    }
+
+    &--1 .stars__star:nth-child(-n+1),
+    &--2 .stars__star:nth-child(-n+2),
+    &--3 .stars__star:nth-child(-n+3),
+    &--4 .stars__star:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-accent;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    margin-bottom: 24px;
+    align-items: flex-end;
+    justify-content: space-between;
+
+    &-text {
+      margin: 0;
+      padding: 0;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
```

Comment:
```suggestion
```

Reply:
I've gettig this as result of `npm test`
``` npm test

> product-cards@1.0.0 test C:\Users\Viktoriya\projects\layout_product-cards
> npm run lint && npm run test:only


> product-cards@1.0.0 lint C:\Users\Viktoriya\projects\layout_product-cards
> mate-scripts lint

✔ Found 1 files
✔ Files analyzed

✨  There's no problem, good job 👏

--- bemlint ---

✨✨ Your BEM is fine! ✨✨


> product-cards@1.0.0 test:only C:\Users\Viktoriya\projects\layout_product-cards
> mate-scripts test

BackstopJS v5.0.1
Loading config:  C:\Users\Viktoriya\projects\layout_product-cards\backstopConfig.js

COMMAND | Executing core for "test"
createBitmaps | Selected 2 of 2 scenarios.
Cookie state restored with: []
Cookie state restored with: []
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
SCENARIO > Link with data-qa_hover
SCENARIO > Card with data-qa_card
Puppeteer encountered an error while running scenario "Link with data-qa_hover"
TypeError: page.waitForTimeout is not a function
x Close Browser
      COMMAND | Executing core for "report"
   See: C:\Users\Viktoriya\projects\layout_product-cards\backstop_data\bitmaps_test\20240214-130206\failed_diff_Link_with_data-qa_hover.png
   See: C:\Users\Viktoriya\projects\layout_product-cards\backstop_data\bitmaps_test\20240214-130206\failed_diff_Card_with_data-qa_card.png
      compare | ERROR { requireSameDimensions: false, size: isDifferent, content: 43.94%, threshold: 1% }: Link with data-qa_hover Link_with_data-qa_hover.png
      compare | ERROR { requireSameDimensions: false, size: isDifferent, content: 1.11%, threshold: 1% }: Card with data-qa_card Card_with_data-qa_card.png
       report | Test completed...
       report | 0 Passed
       report | 2 Failed
       report | Writing browser report
       report | Resources copied
       report | Copied jsonp report to: C:\Users\Viktoriya\projects\layout_product-cards\backstop_data\html_report\config.js
       report | Copied json report to: C:\Users\Viktoriya\projects\layout_product-cards\backstop_data\bitmaps_test\20240214-130206\report.json
      COMMAND | Executing core for "openReport"
   openReport | Attempting to ping 
   openReport | Remote not found. Opening backstop_data\html_report\index.html
       report | *** Mismatch errors found ***
      COMMAND | Command "report" ended with an error after [0.262s]
      COMMAND | Error: Mismatch errors found.
                    at C:\Users\Viktoriya\projects\layout_product-cards\node_modules\backstopjs\core\command\report.js:189:17
                    at processTicksAndRejections (internal/process/task_queues.js:95:5)
      COMMAND | Command "test" ended with an error after [11.367s]
      COMMAND | Error: Mismatch errors found.
                    at C:\Users\Viktoriya\projects\layout_product-cards\node_modules\backstopjs\core\command\report.js:189:17
                    at processTicksAndRejections (internal/process/task_queues.js:95:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! product-cards@1.0.0 test:only: `mate-scripts test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the product-cards@1.0.0 test:only script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Viktoriya\AppData\Roaming\npm-cache\_logs\2024-02-14T18_02_18_541Z-debug.log
npm ERR! Test failed.  See above for more details.```

Reply:
@maxim2310